.github/workflows/install-sfdx-package.yml

```
@@ -244,3 +252,24 @@ jobs:
 
       - name: install package
         run: sf package install --package "${package}" --installation-key "${installation_key_decrypted}" --apex-compile package --upgrade-type Mixed --wait 15
+
+      - name: check out code from specified repository
+        if: ${{ inputs.repo != '' && inputs.commit != '' }}
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ inputs.repo }}
+          ref: ${{ inputs.commit }}
```

We can just use the default options since they do what we want: checkout the branch for which the workflow was triggered of the repository it was triggered from.

```suggestion
      - name: check out code
        uses: actions/checkout@v4
```